Makefile

```diff
@@ -0,0 +1,39 @@
+SHELL := bash
+.ONESHELL:
+.SHELLFLAGS := -eu -o pipefail -c
+.DELETE_ON_ERROR:
+MAKEFLAGS += --warn-undefined-variables
+MAKEFLAGS += --no-builtin-rules
+.DEFAULT_GOAL := help
+
+install:  ## install all dependencies locally
+	poetry install
+.PHONY: install
+
+update:  ## update project dependencies locally (run after git update)
+	poetry update
+.PHONY: update
+
+ci: codespell lint test ## Run all checks (codespell, lint, test)
+.PHONY: ci
+
+test:  ## Run tests
+	poetry run pytest . --no-cov
```

Comment:
pytest failing locally here - looks like I need mongodb installed locally for tests

Reply:
yup needed to install mongodb4.2 locally now local tests are working